stable/graylog/templates/configmap.yaml

```diff
@@ -196,4 +200,8 @@ data:
       -Dgraylog2.installation_source=docker \
       ${GRAYLOG_HOME}/graylog.jar \
       server \
+    {{- if or (.Values.graylog.elasticsearch.secret.enable) (.Values.graylog.mongodb.secret.enable) }}
+      -f ${GRAYLOG_HOME}/graylog.conf.subst
+    {{- else }}
```

Comment:
Why you create a new config file? The proper graylog.conf file should be created directly from configMap

Reply:
I'm creating a new file because graylog.conf is read-only and [here](https://github.com/helm/charts/pull/20807/files#diff-cf2d86cc59c6633d339b43cd57855ccaR191) I'm replacing all occurrences of ${ANYTHING} by the actual value from the env var. Do you have a better way to do that?

Reply:
Ok, I forgot that the file is readonly and composing a new file from scratch is not the good choice for this situation.

Reply:
So I assume you're okay with my solution?

Reply:
it is not the best but it is acceptable for now.
Usually these value are not confidentials but i think you have situations that you need to define these endpoints in secret. I will accept the PR

Reply:
@KongZ Thanks for accepting the PR. As a matter of fact, the values are confidential in my case and hopefully in more cases in the future, when people start using k8s service catalogs like the case with [aws-service-broker](https://github.com/awslabs/aws-servicebroker) and the [mongodb-service-broker](https://github.com/mongodb/mongodb-atlas-service-broker).
[ServiceBinding](https://svc-cat.io/docs/resources/#servicebinding) puts the credentials in a k8s secret for your to consume.